requirements/core.txt

```diff
@@ -12,4 +12,5 @@ pydantic>=1.9.0
 PyYAML
 boto3==1.23.1
 requests
-dask
\ No newline at end of file
+dask
```

Comment:
why is dask added to the core requirements? is it needed here?

Reply:
sorry, I realized that it was already there. got confused with dash.  looks good